src/components/FlatParameters/FlatParameters.js

```diff
@@ -27,6 +28,8 @@ const useStyles = makeStyles((theme) => ({
         margin: 0,
     },
     paramListItem: {
+        display: 'flex',
+        flexDirection: 'row',
```

Comment:
'row': this is the default value for flexDirection then you don't need to set it here.

Reply:
done